src/ucp/core/ucp_ep.c

```diff
@@ -2102,14 +2102,15 @@ void ucp_ep_invoke_err_cb(ucp_ep_h ep, ucs_status_t status)
         /* error callback is not set */
         (ucp_ep_ext_gen(ep)->err_cb == NULL) ||
         /* the EP has been closed by user */
-        (ep->flags & UCP_EP_FLAG_CLOSED)) {
+        (ep->flags & (UCP_EP_FLAG_CLOSED|UCP_EP_FLAG_ERR_HANDLER_INVOKED))) {
```

Comment:
looks like wrong alignment
should it be:
`(ep->flags & (UCP_EP_FLAG_CLOSED | UCP_EP_FLAG_ERR_HANDLER_INVOKED))) {`
or
```
(ep->flags & (UCP_EP_FLAG_CLOSED |
              UCP_EP_FLAG_ERR_HANDLER_INVOKED))) {